Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -1,54 +1,32 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <AssemblyName>Expecto</AssemblyName>
-    <Version>1.1.2</Version>
-    <OutputType>Library</OutputType>
     <TargetFramework>netstandard1.6</TargetFramework>
-    <DebugType>pdbonly</DebugType>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
+    <DefineConstants Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
+      NETSTANDARD1_6;
+      RESHAPED_REFLECTION;
+      @(DefineConstants);
+    </DefineConstants>
   </PropertyGroup>
+  
   <ItemGroup>
     <Compile Include="..\paket-files\logary\logary\src\Logary.Facade\Facade.fs" />
+    <Compile Include="..\Expecto\Performance.fs" />
     <Compile Include="..\Expecto\Expecto.fs" />
     <Compile Include="..\Expecto\Expect.fs" />
   </ItemGroup>
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161104-2</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="FSharp.Core">
-      <Version>4.0.1.7-alpha</Version>
-    </PackageReference>
-    <PackageReference Include="Argu">
-      <Version>3.3.0</Version>
-    </PackageReference>
-    <PackageReference Include="System.Linq.Parallel">
-      <Version>4.3.0</Version>
-    </PackageReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <PackageReference Include="NETStandard.Library">
-      <Version>1.6.1</Version>
-    </PackageReference>
-    <PackageReference Include="FSharp.NET.Sdk">
-      <Version>1.0.0-alpha-000009</Version>
-    </PackageReference>
-    <PackageReference Include="System.Diagnostics.TraceSource">
-      <Version>4.3.0</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161104-2</Version>
-    </PackageReference>
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="dotnet-compile-fsc">
-      <Version>1.0.0-preview2-020000</Version>
-    </DotNetCliToolReference>
+    <!-- Allow intellisense https://github.com/dotnet/netcorecli-fsc/wiki/.NET-Core-SDK-rc4#ide-support -->
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.0-beta-060000" PrivateAssets="All" />
+    <DotNetCliToolReference Include="dotnet-compile-fsc" Version="1.0.0-preview2-020000" />
+    
+    <PackageReference Include="FSharp.Core" Version="4.1.*" />
```

Comment:
@haf there is a wip (used in [fable-compiler/fable-suave-scaffold](https://github.com/fable-compiler/fable-suave-scaffold/blob/master/src/Server/Server.fsproj#L29) ) but ihmo is better merge this pr and after add paket support with another, otherwise you delay a lot because is wip